user-service/index.js

```diff
@@ -21,11 +21,12 @@ const router = express.Router();
 
 // Controller will contain all the User-defined Routes
 // router.get('/', (_, res) => res.send('Hello World from user-service'));
-router.get('/', authenticateToken);
+// router.get('/', authenticateToken);
+router.get('/', authenticateCookieToken);

```

Comment:
I think it's better that you use a unique endpoint instead of `/` as usually for rest apis - `GET  /user` mean you're trying to get all the users in the DB

Reply:
Thanks for your input, I agree with you! Good news is, that also happens to be an unused route from back when the design choices for authentication weren't as sound. Authentication is now done via a callback function in protected endpoints. I have removed this safely in a subsequent commit 😊